inst/templates/Package-DESC.dcf

```diff
@@ -9,10 +9,10 @@ Description: Collection, correction, and connection.
 License: CC BY 4.0
 Depends: 
     R (>= 3.5.0),
-    tibble,
-    pointblank
 Imports:
     manydata,
+    tibble,
+    pointblank,
```

Comment:
If pointblank just for testing, then Suggests? If tibble required for printing, just rely on manydata Depends?